blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubServerContainer.java

```diff
@@ -9,7 +9,8 @@
 import io.jenkins.blueocean.commons.ServiceException;
```

Comment:
@i386 we do not have use case to look for endpoint by URL or name, so not really needed. All thats needed for us is to list all available server endpoints.